CHANGELOG.md

```diff
@@ -29,6 +29,18 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+
+## 0.37.0
```

Comment:
FYI, will be published here: https://github.com/up42/documentation-hub/pull/1144/files